### ISSUE TITLE
License

### ISSUE THREAD

**Issue by Aspie96** [AliaksandrSiarohin/first-order-model#46]
Hi.

I see a license was added in #12.

The added license is not advised for software (Creative Commons iteslf advises not to use it for software). Additionally, it's a proprietary license.

I advise you to opt for Apache 2.0 or a similar open source software license, as advised by Creative Commons. This would make the project maximally useful for others.

**Comment by AliaksandrSiarohin** [comment]
Hi, 
Thank you for your suggestion. 
Unfortunately the choice of license highly depend on my university, so I used the license that my university allow me to use.

**Comment by Aspie96** [comment]
Hi.

Understandable.

This is not relevant here, so I apologize. If you are allowed, would you consider an explicit license for your other projects as well? Any license is less restrictive than no license, because licenses can be rejected.

> Unfortunately the choice of license highly depend on my university, so I used the license that my university allow me to use.

Do you believe it would be possible to build an argument to make your university more flexible on licensing?
This could really help adoption of your project (which from the examples looks amazing). Without a free license (or open source, which is about the same), it will be almost unusable, leading to unofficial clones being more used than your original.
The same applies to pretty much every other software program and that is why open source is far better for academic projects, as it makes research useful for the public.
If you are not very familiar with software licenses, or arguments for free software, I'd be more than willing to help if I can.

**Comment by WilliamOConnell** [comment]
Yeah, do you know why they insisted on a CC license? It seems somewhat nonsensical for software, and I doubt it will actually have whatever the desired effect was.

Nothing against you, I get that you didn't have a choice.

**Comment by AliaksandrSiarohin** [comment]
The source code presented here is just a part of the paper, so to me it make sence that it has the same license as a paper. I saw many other papers publish their source code with the same license. So I belive suggestion from my University is valid here.

**Comment by Aspie96** [comment]
> The source code presented here is just a part of the paper, so to me it make sence that it has the same license as a paper. I saw many other papers publish their source code with the same license. So I belive suggestion from my University is valid here.

Releasing source code and paper under the same license is actually a terrible idea.
That's because they are used for different purposes.

While the paper is only useful for academic purposes, the source code can be used in other fields.
The reason this is important is that it makes research actually useful: we often say that research should get more fundings, but that only makes sense if it contributes to society.
A non-open or non-free license does not contribute to society, unfortunately. It locks research away from everything else.

**Comment by roodrallec** [comment]
Either way, it appears it's now being used as an app:
https://apps.apple.com/us/app/avatarify-ai-face-animator/id1512669147
